src/components/LocationForm.js

```diff
@@ -1,28 +1,52 @@
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 
+// This looks good. A couple of notes
+//  1. React is deprecating the use of refs as a string, so instead, use their function callback
+//  2. bind submit in the constructor
+//  3. always declare any props so any engineer knows what a component needs in order to work
 export default class LocationForm extends Component {
 
-    submit(e) {
-        e.preventDefault();
+  static defaultProps = {
```

Comment:
So this was a typo mistake. This should be `static propTypes` but there is also a secondary `static defaultProps` that allows you to define default values for our properties, especially for default values.